Cargo.toml

```diff
@@ -96,14 +96,14 @@ inventory = { version = "0.3.17" }
 rkyv      = { version = "=0.8.8" }
 
 # Must be pinned with the same swc versions
-swc                 = { version = "=13.0.1" }
+swc                 = { version = "=14.0.0" }
```

Comment:
can we use the latest 14.0.1, see https://github.com/vercel/next.js/pull/76185

Reply:
Done~